Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml

```diff
@@ -40,6 +40,49 @@
   - type: ExplosionResistance
     damageCoefficient: 0.9
 
+#Security Basic armor vest
+- type: entity
+  parent: ClothingOuterBaseMedium
+  id: ClothingOuterArmorBasic
+  name: armor vest
+  description: A standard Type I armored vest that provides decent protection against most types of damage.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Vests/security.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Vests/security.rsi
+  - type: Armor #Based on /tg/ but slightly compensated to fit the fact that armor stacks in SS14.
+    modifiers:
+      coefficients:
+        Blunt: 0.70
+        Slash: 0.70
+        Piercing: 0.70 #Can save you, but bullets will still hurt. Will take about 10 shots from a Viper before critting, as opposed to 7 while unarmored and 16~ with a bulletproof vest.
+        Heat: 0.80
+  - type: ExplosionResistance
+    damageCoefficient: 0.90
+
+#Security Alternate / slim basic armor vest
+- type: entity
+  parent: ClothingOuterArmorBasic
+  id: ClothingOuterArmorBasicSlim
+  name: armor vest
+  suffix: slim
+  description: A slim Type I armored vest that provides decent protection against most types of damage.
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/Vests/security_slim.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/Vests/security_slim.rsi
+  - type: Armor #Based on /tg/ but slightly compensated to fit the fact that armor stacks in SS14.
+    modifiers:
+      coefficients:
+        Blunt: 0.70
+        Slash: 0.70
+        Piercing: 0.70 #Can save you, but bullets will still hurt. Will take about 10 shots from a Viper before critting, as opposed to 7 while unarmored and 16~ with a bulletproof vest.
+        Heat: 0.80
+  - type: ExplosionResistance
+    damageCoefficient: 0.90
```

Comment:
Why are these identical except for sprite? Parent one off the other and leave it be.